lib/shared/messages.js

```diff
@@ -200,17 +200,17 @@ module.exports = {
 
     driveUnplugged: () => {
       return [
-        'Looks like Etcher lost access to the drive.',
+        'Looks like Etcher lost access to the target.',
         'Did it get unplugged accidentally?',
-        '\n\nSometimes this error is caused by faulty readers that don\'t provide stable access to the drive.'
+        '\n\nSometimes this error is caused by faulty readers that don\'t provide stable access to the target.'
```

Comment:
Here, in the second mention, "target" seems awkwardly unspecific, maybe "target" -> "target device"?